api/swagger/swagger.yaml

```diff
@@ -20,7 +20,124 @@ produces:
 paths:
   /swagger:
     x-swagger-pipe: swagger_raw
-
+  
+  # Group 1
+  /entities:
+    x-swagger-router-controller: entities
+    get:
+      description: Retruns list of all entities
+      operationId: getAllEntities
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Entity'
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+    post:
+      description: Add new entities to DB
+      operationId: addEntities
+      parameters:
+        - name: apiKey
+          in: query
+          description: API key
+          required: true
+          type: string
+        - name: body
+          in: body
+          description: entity data
+          required: true
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/UpdateEntity'
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Entity'
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+      
+  
+  /entities/{id}:
+    get:  
+      description: Retruns entity with specified ID
+      operationId: getEntityByID
+      parameters:
+        - name: id
+          in: path
+          description: ID of entity
+          required: true
+          type: integer
+      responses:
+        "200":
+          description: Success
+          schema:
+            $ref: '#/definitions/Entity'
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+    put:
+      description: Update entity
+      operationId: updateEntity
+      parameters:
+        - name: apiKey
+          in: query
+          description: API key
+          required: true
+          type: string
+        - name: id
+          in: path
+          description: ID of entity
+          required: true
+          type: integer
+        - name: body
+          in: body
+          description: entity data
+          required: true
+          schema:
+            $ref: '#/definitions/UpdateEntity'
+      responses:
+        "200":
+          description: Success
+          schema:
+            $ref: '#/definitions/Entity'
```

Comment:
Does this mean we return the updated data ?
If so I don't think it is necessary .We shall return only the Status Message